src/lib/Scenes/City/Components/BMWEventSection/index.tsx

```diff
@@ -1,57 +1,67 @@
 import { Box, color, Sans, Serif } from "@artsy/palette"
 import { CaretButton } from "lib/Components/Buttons/CaretButton"
+import SwitchBoard from "lib/NativeModules/SwitchBoard"
 import { Event } from "lib/Scenes/City/Components/Event"
 import React from "react"
 
 export interface Props {
   title: string
   data: any
-  cityName: string
+  sponsoredContent: { introText: string; artGuideUrl: string }
 }
 
-const renderEvents = events => {
-  return events.map((event, i) => {
-    if (i < 2) {
-      return (
-        <Box key={i} mb={1}>
-          <Event event={event} />
-        </Box>
-      )
-    }
-  })
-}
+export class BMWEventSection extends React.Component<Props> {
+  renderEvents = events => {
+    return events.map((event, i) => {
+      if (i < 2) {
+        return (
+          <Box key={i} mb={1}>
+            <Event event={event} />
+          </Box>
+        )
+      }
+    })
+  }
 
-const getArtGuidePressed = () => {
-  // FIXME: link to art guide
-  console.log("getArtGuidePressed")
-}
+  getArtGuidePressed = artGuideUrl => {
+    SwitchBoard.presentNavigationViewController(this, artGuideUrl)
+  }
+
+  viewAllBmwShows = () => {
+    // FIXME: link to all BMW shows
+    console.log("viewAllBmwShows")
+  }
 
-export const BMWEventSection: React.SFC<Props> = (props: Props) => {
-  const { data, cityName } = props
-  return (
-    <>
-      <Box my={2} px={2}>
-        <Serif size="8">{props.title}</Serif>
-      </Box>
-      <Box mb={2} px={2}>
-        <Sans weight="medium" size="3">
-          Presented by BMW
-        </Sans>
-        <Box mt={1}>
-          <Serif size="3" color={color("black60")}>
-            {`The BMW Art Guide is your go-to-guide to discover private collections of modern and contemporary art which are accessible to the public. Please see below for collections in ${cityName}.`}
-          </Serif>
+  render() {
+    const {
+      data,
+      sponsoredContent: { introText, artGuideUrl },
+    } = this.props
+    return (
+      <>
+        <Box my={2} px={2}>
+          <Serif size="8">{this.props.title}</Serif>
+        </Box>
+        <Box mb={2} px={2}>
+          <Sans weight="medium" size="3">
+            Presented by BMW
+          </Sans>
+          <Box mt={1}>
+            <Serif size="3" color={color("black60")}>
+              {introText}
```

Comment:
Yep, so implicit in the way this is all set up now is that these sponsored content text blurbs do not contain any markup. Judging from the comps that seems like a safe assumption for now, but wanted to call it out.

Reply:
👍